modules/vnee/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/vnee",
+	"url": "https://www.dndbeyond.com/sources/dnd/vnee",
```

Comment:
Redirects to `https://www.dndbeyond.com/sources/dnd/vnee/vecna-nest-of-the-eldritch-eye`